.github/workflows/ci.yaml

```diff
@@ -135,6 +135,9 @@ jobs:
             - name: "Add PHPUnit matcher"
               run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
 
+            - name: "No Plugins"
+              run: composer global config --no-plugins allow-plugins.symfony/flex false
```

Comment:
I think we DO want Flex enabled - iirc, we install it to get a performance boost.